clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/config/GoogleConfigurationProperties.groovy

```diff
@@ -30,6 +30,7 @@ class GoogleConfigurationProperties {
     String accountType
     String project
     boolean alphaListed
+    boolean httpLoadBalancingEnabled // TODO(jacobkiefer): Feature flag for L7 development.
```

Comment:
I agree that being able to compare behavior with/without this enabled will likely be useful during development.